internal/app/app_test.go

```diff
@@ -46,10 +46,13 @@ func TestWait(t *testing.T) {
 		applyArgs    []string
 		noMethodCall bool
 
-		wantErr bool
+		wantAuthRejected bool
```

Comment:
You don’t assert on this one, you are only using it as an input as far as I can see. So I would avoid the `want` prefix and link it to the attributes above.